cps-global-lib/src/main/resources/org/jenkinsci/plugins/workflow/cps/global/UserDefinedGlobalVariable/help.jelly

```diff
@@ -0,0 +1,4 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core">
+  <j:out value="${it.helpHtml}"/>
```

Comment:
All right, I'll do `MarkupFormatter` and probably name those files as just `*.txt`.